src/client/utils.js

```diff
@@ -0,0 +1,7 @@
+// log to Google Analytics
+export default function LogEvent(label) {
+  window.gtag("event", "Navigate", {
+    event_category: "Unemployment",
```

Comment:
Nit: "Unemployment" seems a bit broad, but I guess they're not using these yet anyway. I don't really have a better suggestion.

Reply:
They're actually using a couple dozen event names, but nothing related to unemployment yet! I agree it's too broad but I didn't want to make it too specific to this new page either, seems like the labels should be specific.